apps/auth/views.py

```diff
@@ -39,11 +43,13 @@ class LogoutView(MethodView):
     methods = ["GET"]
     template_name = "auth/logout.html"
 
-    @login_required
     def get(self):
```

Comment:
@login_required did the same thing, so that's my bad that i mentioned that. Please revert this change